test/fixtures/functionMsk/serverless.yml

```diff
@@ -0,0 +1,17 @@
+service: service
+
+configValidationMode: error
+
```

Comment:
We also now add `frameworkVersion: "*"` to test fixtures, to ensure no warning logs of it being missing are exposed